### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by hernando**
Added Python tutorial.

### CODE REVIEW

**Comment by hernando** [comment]
The keyv SHA is still not up to date.

File: doc/feature/feature.md [hunk 1]
```diff
@@ -1,8 +0,0 @@
-Feature specifications {#FeatureSpecs}
-======================
-
-* @subpage compartmentReport
-* @subpage compartmentReportMPI
-* @subpage morphology11
-* @subpage spikeReport
-* @subpage brainSynapses
```
Comment (eile) [inline doc/feature/feature.md]: why removed?

Comment (hernando) [inline doc/feature/feature.md]: Because it was duplicated in a file with a different name. However I'm going to put it back.

**Comment by tribal-tec** [comment]
Shall we try to get brain.py built on OSX? Now it's not configured:
```
Brion [1.8.0-66-g417bccc] without BBPTestData OpenMP Sphinx
-- Not configured brain.py: Boost.Python, Python or Numpy not found
```

**Comment by hernando** [comment]
Working on it, I was looking for the brew packages.

**Comment by hernando** [comment]
The Python wrapping (with Python 2.7) is now also built in Travis.

**Pull request merged**